lighthouse-cli/test/fixtures/dobetterweb/promise_polyfill.js

```diff
@@ -0,0 +1,257 @@
+/*
+ * @license
```

Comment:
@brendankenny suggestions on getting this loaded by the static server? just add static check to fetch this from third party?
and i wish we could just do the npm version but this polyfill won't work on its own since chrome already defines Promise so I modified it, we could try to find another one that storms over it instead?